src/ast/codegen_llvm.cpp

```diff
@@ -658,6 +658,15 @@ void CodegenLLVM::visit(Call &call)
     b_.CreatePerfEventOutput(ctx_, perfdata, sizeof(uint64_t));
     b_.CreateLifetimeEnd(perfdata);
     expr_ = nullptr;
+    b_.CreateRet(ConstantInt::get(module_->getContext(), APInt(64, 0)));
+
+    // create an unreachable basic block for all the "dead instructions" that
+    // may come after exit()
+    Function *parent = b_.GetInsertBlock()->getParent();
+    BasicBlock *unreach = BasicBlock::Create(module_->getContext(),
+                                             "unreach",
+                                             parent);
```

Comment:
Could you explain why this is necessary in the comment?

Reply:
updated it @danobi